app/src/common/shared/org/mozilla/vrbrowser/ui/keyboards/ChinesePinyinKeyboard.java

```diff
@@ -50,7 +50,9 @@ public CandidatesResult getCandidates(String aComposingText) {
         }
 
         // Autocomplete when space is clicked
```

Comment:
nit: We should change this comment because we also do the same things on comma and period.

Reply:
Fixed